qiskit/visualization/latex.py

```diff
@@ -228,13 +216,20 @@ def _initialize_latex_array(self):
             if isinstance(self.ordered_bits[i], Clbit):
                 if self.cregbundle:
                     reg = self.bit_locations[self.ordered_bits[i + offset]]["register"]
-                    self._latex[i][0] = "\\lstick{" + reg.name + ":"
+                    self._latex[i][0] = (
+                        "\\nghost{Register: " + reg.name + "} & " + "\\lstick{" + reg.name + ":"
```

Comment:
Could you let me know if there is any particular reason why you use "Register" here?

Reply:
No real reason. Just needed some tabbing space, and putting in a random placeholder (like XYZAB) would probably be confusing. Instead a word like "Register This" is at least conveying some info. I'm sure there is a better workaround, so any ideas are welcome.

Reply:
This works great in the notebooks. But I think it causes a teeny bit of spacing troubles when you try to take the latex_source code and compile it separately. For instance, the circuit

```python
qc = QuantumCircuit(2)
qc.h([0])
qc.cx(0,1)
```
gives something like 
![image](https://user-images.githubusercontent.com/51048173/120778005-7f0d7080-c543-11eb-8f43-404ea793bddf.png)

and

![image](https://user-images.githubusercontent.com/51048173/120778309-d1e72800-c543-11eb-99ce-bfa0bf3c1ae7.png)


It leaves a lot of space in front. So a suggestion is that you could simply use the parameter of \lstick as the parameter of \nghotst too. That might ensure enough spacing. And you could do similarly for the case when initial_state=True.

Reply:
Good idea, I will try it out.